python/taichi/_main.py

```diff
@@ -230,15 +224,40 @@ def on_mouse_click_callback(example_name):
     @register
     def example(self, arguments: list = sys.argv[2:]):
         """Run an example by name (or name.py)"""
+        def colormap(name):
+            from colorsys import hls_to_rgb  # pylint: disable=C0415
```

Comment:
does this introduce a new dependency for users?

Reply:
`colorsys` is a python built-in module, hence won't introduce new dependency here~